src/com/mostc/pftt/main/CmpReport.java

```diff
@@ -562,18 +540,8 @@ public static void main(String[] args) throws Exception {
 		//File base_dir = new File("C:\\php-sdk\\PFTT-Auto\\PHP_Master-Result-Pack-rd515455-TS-X64-VC11");
 		//File test_dir = new File("C:\\php-sdk\\PFTT-Auto\\PHP_Master-Result-Pack-r04fcf6a-TS-X64-VC11");
 		
-		File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_5-Result-Pack-5.5.10RC1-TS-X86-VC11"));
-		File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_5-Result-Pack-5.5.10-TS-X86-VC11"));
-		//File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.26RC1-TS-X86-VC9"));
-		//File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.26-TS-X86-VC9"));
-		
-		//File test_dir = (new File("C:\\php-sdk\\PHP_5_5-Result-Pack-5.5.8-dev-NTS-X64-VC11-SELF"));
-		//File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.22rc1-NTS-X86-VC9-SQLSVR"));
-		//File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.22-NTS-X86-VC9-SQLSVR"));
-		//File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.22-NTS-X86-VC9"));
-		//File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_4-Result-Pack-5.4.23rc1-NTS-X86-VC9"));
-		//File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_Master-Result-Pack-ra0244a6-NTS-X86-VC11"));
-		//File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_6-Result-Pack-5.6.0-50333-NTS-X86-VC11"));
+		File base_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_Master-Result-Pack-ra0244a6-NTS-X86-VC11"));
+		File test_dir = (new File("C:\\php-sdk\\PFTT-Auto\\PHP_5_6-Result-Pack-5.6.0-50333-NTS-X86-VC11"));
```

Comment:
This doesn't seem useful. Current PFTT2 doesn't depend on any hardcoded path. If there's no use of this, please just remove these lines.

Thanks.

Reply:
Okay, will update the PR ASAP.

Reply:
This code is part of a `static main` which is currently unused. I assume it has been used for development/testing purposes, and as such it might make sense to keep it for now. Otherwise we should rid the [whole method](https://github.com/php/pftt2/blob/master/src/com/mostc/pftt/main/CmpReport.java#L500-L607).

For now I have removed the out-commented lines.

Reply:
I guess this method was used to compare some data without running tests, not sure what the use case was. I think it's ok, perhaps would just make sense to put a TODO there and remove it later or convert to some regular pftt option. If there are no other issues or comments, i'd suggest ot merge this.

Thanks.

Reply:
I've added a TODO item, and going to merge.

Thanks!